packages/pebble-shared/svgs/share-with-people.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="1000pt" height="1000pt" viewBox="0 0 1000 1000"><path d="M358.898 80.3c23.301 0 44.5 9.4 60.5 26.4 15.903 17 23.704 39.198 21.903 62.1l-2.2 29.598c-3.3 42.704-39.203 76.403-81.902 76.403s-78.898-33.5-81.898-76.403l-2.2-29.597C271.5 145.898 279 123.6 295 106.699c16-16.898 37.2-26.398 60.5-26.398h3.398m0-79.7h-4.5C260.2.602 185.801 80.7 192.801 174.5L195 204.102c6.102 84.597 76.602 149.796 161.602 149.796S511.699 288.5 518.199 204.102l2.2-29.602C527 80.7 452.898.602 358.898.602zM345 729.2c0-12.097 1-23.898 3.5-35.597H95.3V583.699c0-53.5 49.4-97.3 109.9-97.3h303c51.3 0 94.198 31.3 106.402 73.3l71.699-35.597c-26.2-68.204-96-117.5-178.2-117.5H205c-104.602 0-189.2 79.296-189.2 176.898v132.602c0 31.5 25.5 56.796 56.802 56.796h278.296C347 759 345 744.301 345 729.2zm0 0"/><path d="M879.3 790.5c-28.198 0-53.5 11-72.5 29.2l-173.1-80.9c.198-3.5.6-6.698.6-10.198v-1.403l175.5-86.597C828.399 656.898 852.5 667 879.5 667c57.8 0 104.602-46.8 104.602-104.602 0-57.796-46.801-104.597-104.602-104.597-57.8 0-104.602 46.8-104.602 104.597 0 2.204.204 4.903.204 6.903l-172 84.597c-19-18.796-44.903-30-73.301-30C472 623.898 425 670.7 425 728.5c0 57.8 46.8 104.602 104.602 104.602 25.5 0 48.796-9.204 67.199-24.5L775 891.8v2.898c0 57.801 46.8 104.602 104.602 104.602 57.796 0 104.597-46.801 104.597-104.602C984.102 837.5 937.301 790.5 879.301 790.5zm0 0"/></svg>
```

Comment:
check if person's "head" should be circle or not

Reply:
it is by designed. Discussed it with Pintu.